.mergify.yml

```diff
@@ -121,22 +118,4 @@ pull_request_rules:
       - "#approved-reviews-by>=1"
       - "#changes-requested-reviews-by=0"
       - status-success~=AWS CodeBuild us-east-1
-      - status-success=validate-pr
-  - name: automatic merge of v2 forward merges
```

Comment:
Don't we want auto-merge of FW merges?

Reply:
Of course. Once we get this in - https://github.com/aws/cdk-ops/pull/1009 - it should not be different from any other PR.

Auto approval is configured here - https://github.com/aws/aws-cdk/blob/master/.github/workflows/auto-approve-v2-merge-forward.yml

Something else I missed?